templates/settings.php

```diff
@@ -237,6 +237,35 @@
 			<?php \do_settings_fields( 'activitypub', 'general' ); ?>
 			<?php \do_settings_fields( 'activitypub', 'server' ); ?>
 		</div>
+		<div class="box">
+			<h3><?php \esc_html_e( 'Security', 'activitypub' ); ?></h3>
+			<table class="form-table">
+				<tbody>
+					<?php if ( ! defined( ACTIVITYPUB_AUTHORIZED_FETCH ) ) : ?>
```

Comment:
```suggestion
					<?php if ( ! defined( 'ACTIVITYPUB_AUTHORIZED_FETCH' ) ) : ?>
```
This needs quotes to avoid triggering a fatal error.

Reply:
🫣